io.catenax.essincident/3.0.0/EssIncident.ttl

```diff
@@ -0,0 +1,349 @@
+   samm:exampleValue "+49-123-456789" .
+
+:flagAnonymous a samm:Property ;
+   samm:preferredName "Flag anonymous"@en ;
+   samm:description "Flag that Incident issuer wants to be anonymous"@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue false .
+
+:essIncidentIssuerId a samm:Property ;
+   samm:preferredName "Issuer ID"@en ;
+   samm:description "System generated unique identifier of incident issuer"@en ;
+   samm:characteristic :IdTrait ;
+   samm:exampleValue "9a47b3c8-b6d4-44f1-99ba-6bdb33916cac" .
+
+:essIncidentIssuerAddress a samm:Property ;
+   samm:preferredName "Ess Incident Issuer Address"@en ;
+   samm:description "Simple form of an address which can belong to a person, organisation, company etc."@en ;
+   samm:characteristic ext-characteristic:PostalAddress .
+
+:essIncidentIssuerLastName a samm:Property ;
+   samm:preferredName "ESS Incident Issuer Last Name"@en ;
+   samm:description "Last name of ESS incident Issuer"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Testuser Last name" .
+
+:essOriginatorCountrySubdivision a samm:Property ;
+   samm:preferredName "ESS Originator Country Subdivision"@en ;
+   samm:description "Region within a country of ESS Originator"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "BR-SP" .
+
+:essIncidentIssuerCountrySubdivision a samm:Property ;
+   samm:preferredName "ESS Incident Issuer Country Subdivision"@en ;
+   samm:description "Region within a country of ESS Incident Issuer"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "IN-AP" .
+
+:IndustryCharacteristic a samm-c:Enumeration ;
+   samm:preferredName "Industry characteristic"@en ;
+   samm:description "Industry Characteristic"@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "Extraction of raw materials" "Manufacture of components / intermediates" "Manufacture of final products" "Distribution / Trade" "Waste treatment / recycling" "Services" "Lending / financing / insurance" "Other" ) .
+
+:IncidentCategoryCharacteristic a samm-c:Enumeration ;
+   samm:preferredName "Incident Category Characteristic"@en ;
+   samm:description "Characteristic for defining an incident category in the context of ESS (Environmental and Social Standards)."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "Environmental" "Social" ) .
+
+:IncidentSubcategories a samm-c:Enumeration ;
+   samm:preferredName "Incident Subcategories"@en ;
+   samm:description "Characteristic for defining subcategories of an incident in the context of ESS (Environmental and Social Standards)."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "Child labour" "Forced labour" "Slavery" "Work safety" "Freedom of association" "Discrimination" "Minimum wage" "Environmental pollution" "Forced eviction" "Force by state security" "Usage of Mercury" "Usage of harmful chemicals" "Non-environmental friendly handling of waste" "Import and export of hazardous waste" ) .
+
+:IdTrait a samm-c:Trait ;
```

Comment:
you can make use of shared aspect uuid here and dont need to define your own trait for uuidv4.
https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/io.catenax.shared.uuid/2.0.0/Uuid.ttl#L35